src/utils/filters.ts

```diff
@@ -1,11 +1,11 @@
 export default {
-    authorizedUrl(urlString) {
-        if (!urlString) {
-            return urlString;
-        }
-
-        var url = new URL(urlString);
-        url.searchParams.set("auth", import.meta.env.VITE_BMM_TOKEN);
-        return url.toString();
+  authorizedUrl(urlString: string | null | undefined) {
+    if (!urlString) {
+      return "";
     }
-}
+
+    const url = new URL(urlString);
+    url.searchParams.set("auth", import.meta.env.VITE_BMM_TOKEN);
```

Comment:
that's not the latest version. it should be `localStorage.getItem("token")`

Reply:
Why does github show the wrong original version?

Reply:
@SimonSimCity Can you try syncing your fork?
Also, in the future you should just create a branch in bcc-code/bmm-web instead of using the fork.

Reply:
Sure, but what you're pointing out here will remain...

Reply:
The file should look like this. I don't get why it's different in your PR.
https://github.com/bcc-code/bmm-web/blob/main/src/utils/filters.ts